yarn.lock

```diff
@@ -3884,7 +3884,7 @@
   dependencies:
     eslint-config-prettier "^7.1.0"
 
-"@stoplight/http-spec@^3.1.1", "@stoplight/http-spec@^3.2.6":
+"@stoplight/http-spec@^3.1.1":
```

Comment:
Can we get rid of this lockfile entry somehow?

Reply:
Same as above. I'll soon send a PR trying to fix this but it's just noise, shouldn't bother us for this PR.